examples/semi_supervised/plot_label_propagation_digits_active_learning.py

```diff
@@ -70,7 +70,9 @@
         lp_model.label_distributions_.T)
 
     # select five digit examples that the classifier is most uncertain about
-    uncertainty_index = uncertainty_index = np.argsort(pred_entropies)[-5:]
+    uncertainty_index = np.argsort(pred_entropies)[::-1]
+    uncertainty_index = uncertainty_index[
+        np.in1d(uncertainty_index, unlabeled_indices)][:5]
```

Comment:
It wouldn't cause an error, but we might as well break the loop earlier.  Also I fixed the charts to handle if the iteration count is increased.